lib/common/util.js

```diff
@@ -360,6 +362,14 @@ function makeAuthenticatedRequestFactory(config) {
     if (stream) {
       return stream;
     }
+
+    return {
+      abort: function() {
+        if (activeRequest_) {
+          activeRequest_.abort();
```

Comment:
Well, let's walk through the scenarios, and keep in mind these things:

- only we see this API, so no users are involved
- `abort()`, which is ultimately from request (the library), doesn't take a callback so it's always just "assumed to have worked"

Our use case is only calling abort when we know an existing HTTP connection is open and we want to kill it; like for this PR, because a subscription object wants to be discarded, but we're still polling for messages.

In the future if we call it erroneously, like we call it 2ms after the connection was closed or something, the abort will just quietly do nothing. But it's all the same to us, because either way, the connection is closed.